src/lib/core/coio.cc

```diff
@@ -680,7 +680,8 @@ coio_service_init(struct coio_service *service, const char *name,
 void
 coio_service_start(struct evio_service *service, const char *uri)
 {
-	if (evio_service_bind(service, uri) != 0 ||
+	const char *uris[] = { uri };
+	if (evio_service_bind(service, uris) != 0 ||
```

Comment:
That is a strange API. So you need to pass count in `init()` but then always pass the same number of uris in `bind`? Maybe better make `bind` take a uri array with its size? I don't see any pros in passing count to `init()`. But I do see that it is quite easy to get problems if you pass an array of wrong size in `bind()` forgetting to update the `init()` call.